lib/tests/streamlit/runtime/app_session_test.py

```diff
@@ -16,10 +16,12 @@
 import threading
 import unittest
 from asyncio import AbstractEventLoop
-from typing import List, Any, Callable, cast, Optional
+from typing import Any, Callable, List, Optional, cast
 from unittest.mock import MagicMock, patch
 
 import pytest
+from tests.isolated_asyncio_test_case import IsolatedAsyncioTestCase
```

Comment:
I think `tests` should be better grouped to the internal imports instead of the third-party ones.